app/views/quick_search/search/_module.html.erb

```diff
@@ -1,15 +1,24 @@
 <div id="<%= service_name.dasherize %>" class="module-contents" >
     <% if searcher.is_a? StandardError %>
+
+        <% if I18n.exists?("#{service_name}_search.loaded_link") %>
```

Comment:
My only suggestion on this pull request is to move this logic to a helper method, and set "module_link" by calling the helper method. It can also be used in _module_with_paging.html.erb.